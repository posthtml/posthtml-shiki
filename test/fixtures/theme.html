<shiki theme="github-light">
  <h1 class="text-xl">Hello</h1>
</shiki>
