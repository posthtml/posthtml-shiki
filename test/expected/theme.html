<pre class="shiki github-light" style="background-color:#fff;color:#24292e" tabindex="0"><code><span class="line"></span>
<span class="line"><span style="color:#24292E">  &#x3C;</span><span style="color:#22863A">h1</span><span style="color:#6F42C1"> class</span><span style="color:#24292E">=</span><span style="color:#032F62">"text-xl"</span><span style="color:#24292E">>Hello&#x3C;/</span><span style="color:#22863A">h1</span><span style="color:#24292E">></span></span>
<span class="line"></span></code></pre>
